apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/networks/index.vue

```diff
@@ -24,13 +24,13 @@ let filters: IFilter[] = reactive([
   {
     title: "Countries",
     refTable: "Countries",
-    columnName: "countries",
+    columnId: "countries",
     columnType: "ONTOLOGY",
     conditions: [],
   },
   {
     title: "Organisations",
-    columnName: "dataCategories",
+    columnId: "dataCategories",
     columnType: "REF_ARRAY",
     refTable: "Organisations",
```

Comment:
should this be refTableId?

Reply:
I am not 100% sure but changed it none the less, we probably need review on catalogue code.